storage/postgres/postgres.go

```diff
@@ -467,7 +467,12 @@ func (p *Postgres) CreateStore(ctx context.Context, store *openfgapb.Store) (*op
 		return nil, handlePostgresError(err)
 	}
 
-	return store, nil
+	createdStore, err := p.GetStore(ctx, store.Id)
+	if err != nil {
+		return nil, err
+	}
+
+	return createdStore, nil
```

Comment:
Could we add a test to prevent this level of inconsistency in the future?

Reply:
Test added. Thank you!